### PR TITLE
Added dutch translation

### PR DESCRIPTION
**Pull request by Hesselni**
Added dutch translation based on feature-i18nSupport reference file 'en.ini'.

### CODE REVIEW

**Comment by marcelfolaron** [comment]
This is awesome! Thank you! I am not quite done adding all translation strings but I'll try to keep new ones to a minimum and add them to the bottom of the file.

File: resources/language/nl.ini [hunk 1]
```diff
@@ -0,0 +1,489 @@
+language.code       = "nl"
+language.direction  = "ltr"
+language.isRTL = "false"
+language.dateformat = "d/m/Y"
```
Comment (marcelfolaron) [inline resources/language/nl.ini]: Slashes can be replaced with hyphens or periods if needed. (so d.n.Y or d-m-Y) is a valid format

File: resources/language/nl.ini [hunk 2]
```diff
@@ -0,0 +1,489 @@
...
+language.jsdateformat = "dd/mm/yy"
+language.timeformat = "h:i A"
```
Comment (marcelfolaron) [inline resources/language/nl.ini]: This is the american timeformat with am/pm. For 24h timeformat you can use:
H:i (https://www.php.net/manual/en/function.date.php)

File: resources/language/nl.ini [hunk 3]
```diff
@@ -0,0 +1,489 @@
...
+language.jstimeformat = "h:mm A"
```
Comment (marcelfolaron) [inline resources/language/nl.ini]: Same as above but in Javascript it's:
HH:mm

File: resources/language/nl.ini (hunk 1 above)
Comment (Hesselni) [reply resources/language/nl.ini]: Good point! I will adjust it.

File: resources/language/nl.ini (hunk 2 above)
Comment (Hesselni) [reply resources/language/nl.ini]: 24h format looks better indeed.

File: resources/language/nl.ini (hunk 3 above)
Comment (Hesselni) [reply resources/language/nl.ini]: Check! I'll adjust it

**Pull request merged**